doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,502 @@
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.50b
+     - `J-Link Software and Documentation Pack`_
+   * - nRF Command Line Tools
+     - v10.6.0
+     - `nRF Command Line Tools`_
+   * - nRF Connect for Desktop
+     - v3.3.0 or later
+     - `nRF Connect for Desktop`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+   * - GCC
+     - See :ref:`Installing the toolchain<gs_installing_toolchain_win>`
+     - `GNU Arm Embedded Toolchain`_
+
+
+As IDE, we recommend to use |SES| (Nordic Edition) version 4.44a.
+It is available from the following links:
+
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x64`_
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`cloud_client` - shows how to connect to and communicate with a cloud service using the generic :ref:`cloud_api_readme`.
+  * :ref:`https_client` - shows how to provision a TLS certificate and connect to an HTTPS server.
+  * :ref:`serial_lte_modem` - demonstrates sending AT commands between a host and a client device.
+    The sample is an enhancement to the :ref:`at_client_sample` sample.
+
+* Added the following libraries:
+
+  * :ref:`lib_aws_iot` - enables applications to connect to and exchange messages with the AWS IoT message broker.
+    The library supports TLS-secured MQTT transmissions and firmware over-the-air upgrades.
+  * :ref:`modem_key_mgmt` - provides functions to provision security credentials to the nRF9160 modem.
+    The library replaces the ``nrf_inbuilt_key`` APIs from the :ref:`nrfxlib:bsdlib`.
+  * :ref:`lib_zzhc` - implements the self-registration functionality that is required to connect to the China Telecom network.
+  * :ref:`supl_client` - integrates the externally hosted SUPL client library.
+    This library implements A-GPS data downloading from a SUPL server.
+
+
+Updated samples and applications
+--------------------------------
+
+* :ref:`asset_tracker`:
+
+  * Added high/low threshold configuration command handling for sensor data. ??? what? Added handling for commands that configure the threshold? Added command handling for threshold configurations?
+  * Modified the command format to match the format that is used by nRF Cloud.
+  * Implemented support for receiving modem AT commands from the cloud and returning the modem's response.
+  * Implemented interval commands from the cloud to the device. ??? what are interval commands? Missing in the documentation.
```

Comment:
The interval is how often sensor data is sent to nRF Cloud.
@jayteemo has the details on this and the threshold configuration

Reply:
Interval, in seconds, at which the device will send sensor data to the cloud. Disabled by setting to 0 or null.

Light Sensor:
`{"appId":"LIGHT", "messageType":"CFG_SET","data": {"interval": 20}}`

Env (temperature, humidity, air pressure, and air quality) Sensor:
`{"appId":"ENV", "messageType":"CFG_SET","data": {"interval": 20}}`

Reply:
Updated to:
  * Added functionality to configure the interval at which sensor data is sent to the cloud.
    This makes it possible to change the poll/send interval for environmental and light sensors from the terminal card in nRF Cloud.